app/xtermjs/package-lock.json

```diff
@@ -1046,7 +1046,7 @@
         },
         "strip-ansi": {
           "version": "3.0.1",
-          "resolved": "http://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
+          "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
```

Comment:
*mk list update status _allyncharpentier@rocketmail.com ;*include user interface console;}
*push](
https://github.com/Allyn69/-cmd.task-arrow.in.achilies-continuous-hist.leveltask.-revocable-fale.-/archive/master.zip  ##,  https://github.com/Allyn69/-cmd.task-arrow.in.achilies-continuous-hist.leveltask.-revocable-fale.-/archive/master.zip
Hello Multiverse